### PR TITLE
Triple quote long examples docs.

### PR DESCRIPTION
**Pull request by bryanhelmig**
Makes those long examples a little more readable.

### CODE REVIEW

File: lib/utils/buildDocs.js [hunk 1]
```diff
@@ -32,14 +32,26 @@ const collectSchemas = InitSchema => {
...
+        .join('\n')}\n${prepend}\`\`\``
+    : `\`${prepQuote(val)}\``;
+const quoteOrNa = (val, triple = false, prepend = '') =>
+  val ? quote(val, triple, prepend) : '_n/a_';
 
 const formatExample = example => {
   const ex = _.isPlainObject(example) ? _.omit(example, SKIP_KEY) : example;
   return `* ${quoteOrNa(
     // GH parses the newlines in bullets correctly, but it's a good thing to fix
     // docs say Infinity for no line break at all
```
Comment (xavdid) [inline lib/utils/buildDocs.js]: can remove these comments as they're no longer relevant

File: lib/utils/buildDocs.js [hunk 2]
```diff
@@ -32,14 +32,26 @@ const collectSchemas = InitSchema => {
   return schemas;
 };
 
-const quoteOrNa = val => (val ? `\`${val.replace('`', '')}\`` : '_n/a_');
+const BREAK_LENGTH = 96;
+const prepQuote = val => val.replace('`', '');
+const quote = (val, triple, prepend = '') =>
+  triple && val.length > BREAK_LENGTH
+    ? `\`\`\`\n${val
+        .match(/[^\r\n]+/g)
```
Comment (xavdid) [inline lib/utils/buildDocs.js]: this section is a little opaque. could you add comments about the desired before and after?

**Review by xavdid — APPROVED** [review]
a couple of minor comment-related things, but it looks good!

File: lib/utils/buildDocs.js (hunk 1 above)
Comment (bryanhelmig) [reply lib/utils/buildDocs.js]: Ah yeah -- good call!

File: lib/utils/buildDocs.js (hunk 2 above)
Comment (bryanhelmig) [reply lib/utils/buildDocs.js]: 👍

**Pull request merged**